support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -184,7 +188,28 @@ function updatePayerStateOrProvince(
   }
   logException('Missing address_state in payment request token and no state/province selected in the form');
   return false;
+}
+
+// Attempt to get country from the token, otherwise fall back on the value in the form
+function updatePayerCountry(
+  countryFromCard?: string,
+  countryFromForm: IsoCountry,
+  setCountry: IsoCountry => void,
+): boolean {
+
+  if (countryFromCard) {
+    const isoCountry = countryFromString(countryFromCard);
+    if (isoCountry) {
+      setCountry(isoCountry);
+      return true;
+    }
+    return false;
```

Comment:
if `countryFromCard` is invalid then we can still use the `countryFromForm`, so I don't think it's necessary to fail.
In fact, since we always have a fallback this function can be:
```
function updatePayerCountry(
  countryFromCard?: string,
  setCountry: IsoCountry => void,
): void
```

Reply:
I am going to get rid of the function and do the update logic inline

Reply:
👍 thanks